monitor_alert_definitions.go

```diff
@@ -0,0 +1,255 @@
+package linodego
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/linode/linodego/internal/parseabletime"
+)
+
+// AlertDefinition represents an ACLP Alert Definition object
+type AlertDefinition struct {
+	ID                int                    `json:"id"`
+	Label             string                 `json:"label"`
+	Severity          int                    `json:"severity"`
+	Type              string                 `json:"type"`
+	ServiceType       string                 `json:"service_type"`
+	Status            string                 `json:"status"`
+	HasMoreResources  bool                   `json:"has_more_resources"`
+	RuleCriteria      *RuleCriteria          `json:"rule_criteria"`
+	TriggerConditions *TriggerConditions     `json:"trigger_conditions"`
```

Comment:
These two should not be null from the API. We can remove the pointers
```suggestion
	RuleCriteria      RuleCriteria          `json:"rule_criteria"`
	TriggerConditions TriggerConditions     `json:"trigger_conditions"`
```